modules/client-configuration/pages/clients-oes.adoc

```diff
@@ -0,0 +1,155 @@
+[[clients-oes]]
+= Registering {oes} Clients
+
+This section contains information about registering clients running {oes} operating systems.
+
+
+
+== Add Software Channels
+
+include::snippets/arch-other-note.adoc[]
+
+Before you register {oes} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The products you need for this procedure are:
+
+[[oes-channels-wizard]]
+[cols="1,1", options="header"]
+.OES Products - WebUI
+|===
+
+| OS Version
+| Product Name
+
+| {oes} 24.4
+| Open Enterprise Server 24.4 x86_64
+
+| {oes} 23.4
+| Open Enterprise Server 23.4 x86_64
+
+|===
+
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[oes-channels-cli]]
+[cols="1,1", options="header"]
+.OES Products - CLI
+|===
+
+| OS Version
+| Base Channel
+
+| {oes} 24.4
+| 
+
+| {oes} 23.4
+| 
+
+|===
```

Comment:
23.4:
"channel_label": "oes23.4-pool-x86_64"
"channel_name": "OES23.4-Pool for x86_64",

24.4:
"channel_label": "oes24.4-pool-x86_64",
"channel_name": "OES24.4-Pool for x86_64",

Reply:
ok